### DISCUSSION TITLE
Production Ready?

### DISCUSSION THREAD

**Discussion by deanfuller43** [bitnami-labs/sealed-secrets#1636] [Q&A]
Within my organisation we have been discussing the state of sealed-secrets and if it's appropriate for a production platform, we're mindful there's components still referenced as alpha. Our testing has been very positive and feedback is good, we're now at the final stage where we would like to recommend it and this is the final outstanding question.

Are others leveraging this in large enterprise production organisations and do you have any guidance? Appreciate any thoughts and feedback.

**Comment by mkmik** [comment]
I think it's time to move the crd version out of alpha. It's been stable for ages now and most changes that happened during the alpha stage were backwards compatible anyway so they would have worked fine even in a `v1` schema.
